benches/src/db_lookup_times_utils/utils.rs

```diff
@@ -0,0 +1,141 @@
+use crate::{
+    db_lookup_times_utils::full_block_table::{
+        BenchDatabase,
+        BenchDbColumn,
+    },
+    utils::ShallowTempDir,
+};
+use anyhow::anyhow;
+use fuel_core::{
+    database::database_description::DatabaseDescription,
+    state::rocks_db::RocksDb,
+};
+use fuel_core_storage::kv_store::{
+    KeyValueInspect,
+    StorageColumn,
+};
+use fuel_core_types::{
+    blockchain::block::{
+        Block,
+        CompressedBlock,
+    },
+    fuel_tx::Transaction,
+    fuel_types::{
+        BlockHeight,
+        ChainId,
+    },
+};
+use itertools::Itertools;
+use rand::{
+    rngs::ThreadRng,
+    Rng,
+};
+use std::path::Path;
+use strum_macros::AsRefStr;
+
+pub type Result<T> = core::result::Result<T, anyhow::Error>;
```

Comment:
Isn't this basically just the same as re-exporting `anyhow::Result`?

```suggestion
pub use anyhow::Result
```

Reply:
Also I guess we don't need to re-export it if we use the anyhow result directly, since it's already a public type.

```suggestion
use anyhow::Result
```

Reply:
we follow this pattern quite a bit in other places in the repository though, will fix.

addressed in a73f83916

Reply:
Oh interesting. I wonder if there is any thought behind it.